backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/validationRule/NamePairPredicates.java

```diff
@@ -91,13 +91,13 @@ public String toString()
 		}
 
 		@Override
-		public ImmutableSet<String> getParameters()
+		public ImmutableSet<String> getParameters(@Nullable final String contextTableName)
 		{
 			ImmutableSet<String> parameters = this._parameters;
 			if (parameters == null)
 			{
 				parameters = this._parameters = predicates.stream()
-						.flatMap(predicate -> predicate.getParameters().stream())
+						.flatMap(predicate -> predicate.getParameters(null).stream())
```

Comment:
i think u used `null` here because the cached parameters shall not be dependent on contextTableName....

if that's the case pls document why you use that 
because first thing i am tempt to say when i see this is that's a bug.